R/pkg/R/functions.R

```diff
@@ -2437,6 +2437,7 @@ setMethod("date_format", signature(y = "Column", x = "character"),
 #'
 #' @param x Column containing the JSON string.
 #' @param schema a structType object to use as the schema to use when parsing the JSON string.
+#' @param asJsonArray indicating if input string is JSON array or object.
```

Comment:
maybe clarifying
`JSON array or object.` -> `JSON array of objects or a single object.`